llm4pddl/approaches/llm_open_loop_approach.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, num_completions: int, temperature: float) -> None:
         self._llm = OpenAILLM(FLAGS.llm_model_name)
         # Set after learning.
         self._prompt_prefix = ""
+        self.embeddings_mapping: List = []
```

Comment:
let's make this a private attribute (`self._embeddings_mapping`). also, "mapping" makes it sound like a dict, but this is a list? also, we'll want to make this type annotation more specific